tests/scheduled_checks.py

```diff
@@ -0,0 +1,81 @@
+"""
+These tests are not called on every push or pull request. They're time-intensive, so on GitHub they're only called
+monthly. To run them locally, use the following command:
+
+    pytest -s -rpP tests/scheduled_checks.py
+
+Because the filename doesn't begin with test_, it's not automatically run with pytest. The -s flag is used to print
+the results of the tests, and the -rpP flags are used to suppress the output of the tests that pass.
+
+"""
+from astroquery.simbad import Simbad
+
+from astrodbkit2.utils import _name_formatter
+
+def test_SIMBAD_resolvable(db):
+    # Verify that all sources have valid coordinates
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Add all IDS to the Simbad output as well as the user-provided id
+    Simbad.add_votable_fields("ids")
+    Simbad.add_votable_fields("typed_id")
+
+    simbad_results = Simbad.query_objects(name_list)
```

Comment:
I think this can be a fixture since it's used by two tests. https://docs.pytest.org/en/6.2.x/fixture.html#back-to-fixtures

```Python
@pytest.fixture
def simbad_results():
    results = db.query()
    simbad_results = Simbad.query
    return simbad_results
```